selfdrive/controls/lib/lateral_mpc_lib/SConscript

```diff
@@ -38,9 +38,7 @@ generated_files = [
   f'{gen}/lat_model/lat_expl_vde_adj.c',
 
   f'{gen}/lat_model/lat_model.h',
```

Comment:
```suggestion
  f'{gen}/lat_model/lat_model.h',
  f'{gen}/lat_cost/lat_cost.h.h',
```

It looks like they moved these three headers into `lat_cost.h` in the update from https://github.com/commaai/openpilot/pull/28821.